README.adoc

```diff
@@ -32,25 +32,25 @@ Here is a quick teaser of a complete Spring Boot application in Java:
 	@SpringBootApplication
 	public class Example {
 
-		@RequestMapping("/")
-		String home() {
-			return "Hello World!";
-		}
+	    @RequestMapping("/")
+	    String home() {
+	        return "Hello World!";
+	    }
 
-		public static void main(String[] args) {
-			SpringApplication.run(Example.class, args);
-		}
+	    public static void main(String[] args) {
+	        SpringApplication.run(Example.class, args);
+	    }
 
 	}
 ----
 
 
 
-== Getting help
+== Getting Help
 Are you having trouble with Spring Boot? We want to help!
 
-* Check the {docs}/html/[reference documentation], especially the {docs}/html/howto.html#howto[How-to's] -- they provide solutions to the most common questions.
-* Learn the Spring basics -- Spring Boot builds on many other Spring projects; check the https://spring.io[spring.io] web-site for a wealth of reference documentation.
```

Comment:
I think this should be `--` so that they're rendered as an em dash.

Reply:
So to be consistent, change the single ‘-‘ to double for line 56?

Reply:
Yes please.